src/components/InstructorStatsCard/index.js

```diff
@@ -14,18 +14,28 @@ export default ({publishedLessons, publishedCourses}) => {
   const items = [
     {
       type: 'lesson',
-      text: `${publishedLessons} published lessons`,
+      text: publishedLessons === 1
```

Comment:
I love util functions like https://www.npmjs.com/package/pluralize to manage this consistently.